charts/minikube-values.yaml

```diff
@@ -106,6 +106,12 @@ graph:
     tokenEncryption:
       secret: MGYxM2JiZTMzY2NhNDg1OQo=
 
+  knowledgeGraph:
```

Comment:
Same here, the minikube deployment should not rely on `dev.renku.ch`. I think you have to leave this empty here and provide the correct value to the helm command more like this: 
`helm  upgrade --install --set knowledgeGraph.services.renku.url = $(minikube ip) ... `.
See also https://github.com/SwissDataScienceCenter/renku/blob/master/charts/README.rst

Reply:
Thanks for that!